docs/documentation_guidelines.md

```diff
@@ -0,0 +1,11 @@
+# Documentation guidelines
+
+1. Each repository should contain an automatically updated index page in `docs` directory.
+2. Repository main `README.md` file should contain a link to the concerned repository index page, where all documentation contents are stored.
+3. Each `.md` file should contain at the beginning a title and a document description in a paragraph below. These two elements of `.md` file will be used to create index page.
+4. A `docs` directory under the repository root doesn't have to contain documentation related strictly to the one tool only. For example we can have a documentation describing a system comprised of multiple tools which together deliver some service for users.
```

Comment:
I think we should first say what it should contain, not what it shouldn't contain :D 

```suggestion
4. A `docs` directory under the repository root doesn't have to contain documentation related strictly to the one tool only. For example, we can have a documentation describing a system comprised of multiple tools which together deliver some service for users.
```